### DISCUSSION TITLE
Idea: RCT Classic Title Screen

### DISCUSSION THREAD

**Discussion by That-Human-Being** [OpenRCT2/OpenRCT2#15751] [Feature requests]
This is just an idea (no clue about how effective it would be, but here it is anyway) but I personaly prefer the title screen from RCT Classic, to the provided options in OpenRCT2. Would it be possible to add it to the base game or an integration? 🤷‍♂️ 

Thanks!

**Comment by duncanspumpkin** [comment]
Feel free to make it using the title sequence editor
